src/Microsoft.AspNet.Http.Extensions/HttpContextExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http.Features;
+
+namespace Microsoft.AspNet.Http
+{
+    public static class HttpContextExtensions
+    {
+        public static async Task<ISession> GetSessionAsync(this HttpContext httpContext)
```

Comment:
Actually, do you need this now that LoadAsync is directly exposed?  `HttpContext.Session.LoadAsync();`